reports/views.py

```diff
@@ -350,3 +357,52 @@ def get(self, request, *args, **kwargs):
                 ]
             )
         return response
+
+
+class EnrolmentReportCsvView(ExportReportCsvView):
+    model = EnrolmentReport
+
+
+@staff_member_required
+@require_http_methods(["POST"])
+def sync_enrolment_reports_view(request):
+    create_from = Enrolment.objects.aggregate(Min("enrolment_time"))[
+        "enrolment_time__min"
+    ]
+    sync_enrolment_reports(hydrate_linkedevents_event=True, create_from=create_from)
+    messages.add_message(request, messages.SUCCESS, _("Enrolment reports synced!"))
+    return HttpResponseRedirect(reverse("admin:reports_enrolmentreport_changelist"))
+
+
+class EnrolmentReportListView(generics.ListAPIView):
```

Comment:
I'm guessing that the Power Bi user will use either `SessionAuthentication` or `BasicAuthentication`?

Reply:
Yep. So do need any changes here? Do we need to narrow the selection?

Reply:
Just confirming that it was the plan. No need to change anything. :+1: